config/locales/en/shared.yml

```diff
@@ -263,6 +263,7 @@ en:
         address: Correspondence address
         age: "%{years} years old"
         age_question: Age at computation date
+        means_tested: Was the client means-tested?
```

Comment:
I do not know if a content person or others have changed the requested text but the ticket specifies the text to be "Does the client need a means test?"

Reply:
It was reviewed on 3711 by content - at the time this is shown it is in the past 👍